pages/docs/adapters/calculix/adapter-calculix-configure.md

```diff
@@ -95,6 +102,8 @@ CalculiX is designed to be compatible with the Abaqus file format. Here is an ex
 
 The adapter internally uses the CalculiX data format for point forces to apply the FSI forces at the coupling interface. This data structure is only initialized for those nodes, which are loaded at the beginning of a CalculiX analysis step via the input file. Thus, it is necessary to load all nodes of the node set, which defines the FSI interface in CalculiX (referring to the above example, the nodes of set "interface" (Note that in CalculiX a node set always begins with an "N" followed by the actual name of the set, which is here "interface".) are loaded via the "CLOAD" keyword.), in each spatial direction. However, the values of these initial forces can (and should) be chosen to zero, such that the simulation result is not affected.
 
+When using faces mesh, instead of a node set (\*NSET), a \*SURFACE must be sent, defined by a list of elements and face numbers. Instead of starting with a "N", the name must start with a "S".
```

Comment:
```suggestion
When using faces mesh, instead of a node set (`*NSET`), a `*SURFACE` must be sent, defined by a list of elements and face numbers. Instead of starting with a `N`, the name must start with a `S`. For example: `SCalculiX_Mesh` (while in the preCICE configuration the name remains `CalculiX_Mesh`).
```